.specify/scripts/bash/check-prerequisites.sh

```diff
@@ -57,13 +57,13 @@ OPTIONS:
 EXAMPLES:
   # Check task prerequisites (plan.md required)
   ./check-prerequisites.sh --json
-  
+
```

Comment:
You have some local IDE settings that mess up the empty files in the `.specify` folder (I guess) - can you exclude those changes from the PR - it is clutter.

Reply:
Will go one more time with this. Spec-kit was most of pain to merge  :) also, for future, please, don't merge develop into PRs, rebase PRs on develop head